src/workerd/server/server.c++

```diff
@@ -3502,6 +3533,75 @@ kj::Own<Server::Service> Server::makeWorker(kj::StringPtr name,
     errorReporter.addError(kj::str("Worker must specify compatibilityDate."));
   }
 
+  kj::Vector<FutureSubrequestChannel> subrequestChannels;
+  kj::Vector<FutureActorChannel> actorChannels;
+  kj::Vector<FutureActorClassChannel> actorClassChannels;
+
+  auto confBindings = conf.getBindings();
+  kj::Vector<WorkerdApi::Global> globals(confBindings.size());
```

Comment:
Does this vector grow? Can we use heapArray?

Reply:
Two lines down, we only add to this vector conditionally if `createBinding()` returns non-null. We don't know the actual size until then, though it usually ends up being `confBindings.size()`.

In any case this code is not new, just moved.